pkg/addons/clusterautoscaler/templates_generate.go

```diff
@@ -0,0 +1,21 @@
+// +build ignore
+
+package main
+
+import (
+	"log"
+
+	"github.com/shurcooL/vfsgen"
```

Comment:
We use bindata in other place, is there any particular reason to use this package here?

Reply:
Didn't know about `bindata`. I used `vfsgen` as that's what is used in Flux and in WKS.